e2e/cypress/support/index.js

```diff
@@ -121,6 +123,13 @@ before(() => {
             timezone: {automaticTimezone: '', manualTimezone: 'UTC', useAutomaticTimezone: 'false'},
         });
 
+        // # Upload license
+        cy.apiGetClientLicense().then(({license}) => {
+            if (license.IsLicensed === 'false' && resetBeforeTest) {
+                cy.apiUploadLicense(serverLicense.filePath);
+            }
+        });
```

Comment:
I suggest to do it if required by test only and not for every test. Good places are inside of `cy.apiRequireLicense` and `cy.apiRequireLicenseForFeature`. On those commands, make a note about the location of license file which should be available in `fixtures` folder. Make the call fix as `cy.apiUploadLicense('mattermost-license.txt')`